recipe/meta.yaml

```diff
@@ -15,7 +15,6 @@ source:
 
```

Comment:
fn: {{ name }}-{{ version }}.{{ compress_type }}  - this can be removed, I think

Reply:
yes, done that. Did not notice initially.